lib/tether/ops_linux.go

```diff
@@ -1072,3 +1079,54 @@ func createBindSrcTarget(files map[string]os.FileMode) error {
 
 	return nil
 }
+
+// LaunchUtility allows for starting, blocking on, and receiving the exit code of a
+// process while in the presence of an embedded child reaper.
+// The function passed in will be launched under lock and MUST NOT wait for the process to
+// exit. It's expected the function be a closure wrapped around StartProcess or similar.
+func (t *BaseOperations) LaunchUtility(fn UtilityFn) (<-chan int, error) {
```

Comment:
this call redirect is a little confusing. can we keep the code inside this method?